api-doc.md

```diff
@@ -309,10 +309,11 @@
     - Headers : none
     - Body :
         {
-            "name": "mansi dhameliya",
-            "date": "3 jan 2023",  // OR if you want to add multiple dates, you can add like this : "3 jan 2023 to 5 jan 2023"
-            "status": 1,  // you can use 1 for request
-            "reason": "Casual Leave",
+            "name":     "mansi dhameliya",
+            "date":     "3 jan 2023",  // OR if you want to add multiple dates, you can add like this : "3 jan 2023 to 5 jan 2023"
+            "duration": "morning" // You can add morning or evening or 3 days as you want
```

Comment:
Shouldn't it be first half, second half instead of morning and evening?

Reply:
Yes, I thought the same, but it really depends on the morning, evening or second half unity, which they will cross the value, this name Pratik told me.